extensions/amp-analytics/analytics-vars.md

```diff
@@ -361,3 +371,75 @@ Provides the number of seconds that have elapsed since 1970. (Epoch time)
 
 Example value: `1452710304312`
 
+## Visibility Variables
+
+### backgrounded
+
+A binary variable with possible values of 1 and 0 to indicate that the page/tab was sent to background at any point before the hit was sent. 1 indicates that the page was backgrounded while 0 indicates that the page has always been in the forground.
+
+### backgroundedAtStart
+
+A binary variable with possible values of 1 and 0 to indicate that the page/tab backgrounded at the time when the page was loaded. 1 indicates that the page was loaded in the background while 0 indicates otherwise.
+
+### continuousVisibleTime
+
+Provides the maximum amount of continuous time an element has met the `visibilitySpec` conditions at the time this ping is sent. Note that a ping with a continuousTimeMin=1000 and totalTimeMin=5000 that is visible for 1000ms, then not visible, then visible
+for 2000ms, then not, then visible for 1000ms, then not, then visible for 1020ms
+will report 2000 for this number as that is the max continuous visible time,
+even if it is not the current continuous visible time (1020 in this example).
+
+### elementHeight
+
+Provides the height of the element specified by `visibilitySpec`.
+
+### elementWidth
+
+Provides the width of the element specified by `visibilitySpec`.
+
+### elementX
+
+Provides the X coordinate of the left edge of the element specified by `visibilitySpec`.
+
+### elementY
+
+Provides the Y coordinate of the top edge of the element specified by `visibilitySpec`.
+
+### firstSeenTime
+
+Provides the time when at least 1px of the element is on the screen for the first time since javascript load.
```

Comment:
Can this be formalized some more. For example, would it be fair to say "since the page became visible"? Or is this really gated on JS load. Does it specifically mean amp-analytics JS?